src/reconcile/rec_write.c

```diff
@@ -6591,3 +6594,37 @@ __rec_dictionary_lookup(
 	*dpp = next;
 	return (0);
 }
+
+/*
+ * __verbose_lookaside_write --
+ *	Create a verbose message to display with details about the transaction
+ * state when performing a lookaside table write.
+ */
+static void
+__verbose_lookaside_write(WT_SESSION_IMPL *session, uint64_t inserted_cnt)
```

Comment:
The usual protocol is for static functions to have names that match the subsystem. All other static functions in this file start with a ``__rec_`` prefix - why is this one different?

Why did you call the parameter ``inserted_cnt`` when the variable in the only calling function is ``insert_cnt``?